pkg/vmimage/data/etc/systemd/system/mdsd.path

```diff
@@ -0,0 +1,8 @@
+[Path]
```

Comment:
it may be less opaque to add systemctl enable(s) / systemctl start(s) to near the bottom of the master and worker startup scripts